.github/workflows/appimage.yml

```diff
@@ -1,25 +1,31 @@
-name: Build AppImage rplidar module
-
-concurrency: 
-  group: ${{ github.workflow }}-${{ github.ref }}
+name: Build AppImage
 
 on:
   workflow_dispatch:
-  push:
-    branches: [ 'main' ]
-    paths-ignore:
-      - 'README.md'
+    inputs:
+      release_type:
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      release_type:
+        required: true
+        type: string
+    secrets:
+      GCP_CREDENTIALS:
+        required: true
 
 jobs:
   appimage:
-    name: AppImage Build Rplidar Module
+    name: AppImage Build
     strategy:
+      fail-fast: false
       matrix:
         include:
-          - arch: [x64, qemu-host]
+          - arch: [ubuntu-latest]
             image: ghcr.io/viamrobotics/canon:amd64-cache
             platform: linux/amd64
-          - arch: [arm64, qemu-host]
+          - arch: [buildjet-2vcpu-ubuntu-2204-arm]
```

Comment:
How long is the build process here? The 2cpu variant isn't very powerful, but we also have a cap on the number of buildjet cpus we can use simultaneously, which we're already maxing out during the busy part of the day.

Reply:
If you're fine with it though, I'm fine with it. As I'm assuming you already tested it, I'm guessing it's not super slow.

Reply:
Discussed offline and James is alright us trying the new 2cpu variant